pygin/pygin/src/panel.cpp

```diff
@@ -4,6 +4,25 @@
 #include "far_api.hpp"
 #include "py.integer.hpp"
 
+namespace {
+
+void WINAPI free_py(void* UserData, const struct FarPanelItemFreeInfo* Info)
+{
+	py::object dec_ref(static_cast<PyObject*>(UserData));
+}
+	
+void let_far_manage_py_items_life_time(std::span<PluginPanelItem> const& items) noexcept
+{
+	for (auto& item : items)
+	{
+		auto* inc_ref = static_cast<PyObject*>(item.UserData.Data);
+		inc_ref = py::object::from_borrowed(inc_ref).release();
```

Comment:
Why do we need to assign the result of `release()` to `inc_ref`? It goes out of scope anyway.

Reply:
To silence the warning about discarding return value... Is there maybe a special attribute syntax for that?

Reply:
`(void)x.release()` should work.

Reply:
Fixed in 53a21210c2de84959c76785e9d52dd10a54852d1.